src/style-dictionary-dist2/TerrawareTheme2.ts

```diff
@@ -0,0 +1,533 @@
+import { PaletteOptions } from '@mui/material/styles/createPalette';
+
+declare module '@mui/material/styles/createPalette' {
+  export interface Palette {
+    tw-clr-base-blue-050?: React.CSSProperties['color'];
```

Comment:
nice!!

Reply:
i like it, too, but it caused the build to fail 😭

Reply:
ok so maybe your original idea of PascalCase is safer..
something like `'tw-clr-base-blue-050'?: React.CSSProperties['color'];` might work.. but we may need to  reference it as such `theme.palette['tw-clr-base-blue-050']`.. `theme.palette.tw-clr-base-blue-050` may not work.
wondering if we can control this with a babel and/or tslint config.

Reply:
fwiw, `-` is interpreted as the `subtraction operator` in `theme.palette.tw-clr-something` .. hence the need to `theme.palette['tw-clr-something']`
maybe best to stick with PascalCase for now - unless we think it's ok to require such usage `theme.palette['abc-def']`?

Reply:
I think sticking with PascalCase seems like the way to go for now 👍